hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -52,8 +52,11 @@
 import org.apache.hadoop.hive.metastore.api.hive_metastoreConstants;
```

Comment:
Two other suggestion for this class: can we add in comment of "HIVE_TABLE_PROPERTY_MAX_SIZE" , one more sentence to let user know how to turn off feature?

```
// set to 0 to not expose Iceberg metadata in HMS properties
```
And also, a precondition in HiveTableOperations constructor to check if  value is non-negative.

Reply:
Added the comment. Negative is fine, right?

Reply:
I think probably better to disallow negative as it makes little sense?  But to me its ok either way.